content/departments/product-engineering/product/process/product_feedback_rotation.md

```diff
@@ -0,0 +1,88 @@
+
+Sometimes, someone outside Sourcegraph will submit a GitHub issue. At Sourcegraph, issues are labelled with team labels, that look like `team/batchers` so that they show up on the dashboard of the right team. All issues labeled with a team's name are [automatically added](https://github.com/sourcegraph/sourcegraph/blob/main/.github/workflows/label-move.yml) to the team's board for triage. Issues without labels tend to get lost. External contributors cannot add labels, so the PM on feedback rotation has to label those issues. To keep the backlog clean, this means you may also have to add a team label to issues created by Sourcegraph teammates that forgot to do it. View it as an opportunity to peak into what other teams are doing!
+
+In practice:
+
+- Monitor the list of [new issues to triage](https://github.com/sourcegraph/sourcegraph/issues?page=2&q=is%3Aissue+no%3Alabel+is%3Aopen)
+- If unsure about what team label to use, take a look at [list of team labels](https://github.com/sourcegraph/sourcegraph/labels?q=team+%2F). The slack channel associated with the team is in the label description. You can refer to the [product teams page](../team/product_teams.md) and to the more detailed [engineering ownership](../../engineering/process/engineering_ownership.md).
+- If you really don't know what team to assign it to, take a shot at assigning to the closest team with a nice message, asking them to re-route it to a more relevant team if necessary. "Hey, this issue has no team label and this _seems_ like it's in your scope. Adding your team label, but feel free to re-route this to another team".
+
+There is no SLA for this, just make sure that all issues are labelled at the end of your rotation.
+
+### Triaging producboard feedback
+
+This is the third and last thing you need to do as the PM on feedback rotation. Feedback from various sources are submitted into productboard, and need to be assigned to the right owner.
+
+Note that there is currently a huge backlog of unsorted notes. You are only responsible with notes submitted during your rotation.
+
+- Go to [Unprocessed notes](https://sourcegraph.productboard.com/insights/notes/unprocessed/notes/21590532)
+- Steps **for all notes**:
+  - Owner assignment: Unassign the Owner, assign whoever should own the insight (PM or EM). Note that in some cases, this happens automatically.
+  - User assignment: Assign User as the Company or create a new user if it hasn’t been assigned by default.
+  - Tags: Add any relevant team tags.
+  - Features: Highlight content and assign any relevant features you find. When in doubt, leave it to the new Owner to triage (PM or EM).
+  - If there is no feedback in the item, it can be archived.
+  - When you are complete with a note that isn’t getting “archived” – please mark it as “processed.”
+
+For SourcegraphWeb notes, there are additional routing rules:
+
+- Browser uninstalls: Assign to Joel Kwartler
```

Comment:
This used to be the case when @Joelkw acted as extensibility PM, maybe we should list Ryan now?

Reply:
Yes please, let's list Ryan!

Reply:
@ryankscott I am adding you here as the owner for extensibility, I assume @Joelkw will hand this over to you in onboarding buddy syncs, but if that's unclear, let me know and we can review that together :)